### DISCUSSION TITLE
FDS - general questions

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/fds#12916] [Q&A]
Hi,

I have a few questions regarding FDS. These are the following:

- I was just wondering if I can model wall/corner tests (i.e., where the burner is against the wall) using FDS. How is a non-axisymmetric plume modelled in FDS?

- I was just wondering how multiple rooms with different ceiling heights are modelled using FDS. 

- Is there a limitation to the size of vent I can use for an FDS scenario?

Thanks.

**Comment by Er9y714** [comment]
1. Yes you can model and you can observe the different plume temperature as expected.
2. Please read User Guide and see how obstuctions are used in the model.
3. Can you elaborate the question please? I recommend reading VENT section in User Guide

**Reply by FireStudent321** [reply]
Thanks for the reply. 

3. At what opening size does the program stop working?

**Reply by Er9y714** [reply]
I am not aware of such a size limitation.

**Reply by FireStudent321** [reply]
Thanks for the reply.

**Comment by mcgratta** [comment]
Validation Guide, NIST/NRC Corner Effects

**Reply by FireStudent321** [reply]
Thanks for the reply.
